AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -109,17 +194,36 @@ public final class PayToComponent: PaymentComponent,
         )
         formViewController.title = paymentMethod.displayInformation(using: configuration.localizationParameters).title
         formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
         formViewController.append(flowSelectionTitleLabelItem.padding())
         formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
         formViewController.append(flowSelectionItem.padding())
+        formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
+        formViewController.append(identifierPickerItem.padding())
+        formViewController.append(FormSpacerItem(numberOfSpaces: 1))
+
+        contentView(formVC: formViewController)
 
         if configuration.showsSubmitButton {
             formViewController.append(FormSpacerItem(numberOfSpaces: 2))
-            formViewController.append(continueButton)
+            formViewController.append(continueButtonItem)
         }
 
         return formViewController
     }()
+
+    // MARK: - Private
+
+    private func contentView(formVC: FormViewController) {
```

Comment:
What are these used for? The function name could be a bit more descriptive

Reply:
It is for constructing UI but I can rename it :)